aikau/src/main/resources/alfresco/services/SiteService.js

```diff
@@ -1182,8 +1182,17 @@ define(["dojo/_base/declare",
             name: "alfresco/forms/controls/MaskingTextBox",
             config: {
                fieldId: "SHORTNAME",
-               mask: "[^0-9a-zA-Z\-\s]",
                targetId: "TITLE",
+               replacements: [
+                  {
```

Comment:
Because of the use of the "trim" attribute - which is an easy to understand option for people that don't know Regular Expressions very well.